src/runtime_src/core/edge/user/aie/aie.cpp

```diff
@@ -32,6 +32,12 @@
 
 namespace zynqaie {
 
+/* Added this lock to make AIE context thread safe. If AIE context is opened
```

Comment:
```suggestion
/* Use a lock to make AIE context thread safe. If AIE context is opened
```

Reply:
Just describe the fact. No need to describe the fact that you added it, since it will not be useful for the readers 20 years from now.